Jenkinsfile

```diff
@@ -13,15 +13,17 @@ def tryStep(String message, Closure block, Closure tearDown = null) {
         }
     }
 }
+
 String BRANCH = "${env.BRANCH_NAME}"
+Boolean IS_TAG = BRANCH ==~ /v(\d{1,3}\.){2}\d{1,3}/
```

Comment:
Minor detail, maybe we can improve the `IS_TAG` constant name since we are actually checking for a valid `semantic version` here which could confuse.